### DISCUSSION TITLE
Everything is a false match

### DISCUSSION THREAD

**Discussion by tsiawd** [mcguirepr89/BirdNET-Pi#1144] [General]
I got bird pie setup and I am having problems with it making false matches. Nothing is the right match most of the birds matched aren't even in this country. I'm using audio from a rtsp feed and I have tried two different cameras and they both produce the same results. I would like to get a proper mic but I'm not sure that's the problem.

**Comment by MueJosh** [comment]
Tools -> Settings
Did you select your Location? 
The "Species Occurence Frequency Threshold" (at the Model selector)?

And under the advanced settings: 
Minimum confidence?

**Reply by tsiawd** [reply]
Yes I put in my location, just changed my species threshold never saw that one minimum confidence is set to 85% now. It keeps thinking crows are cuban pewee 16 times and great horned owls are apperently coming up as orange chinned parakeet 21 times and so on. Funny thing is if I play back the recording to merlin on android it picks everyone of them up 100% pretty much.

**Reply by MueJosh** [reply]
Okay that's weird
I'm not familiar with rtsp streams so I can't really help. 
Have you tried it with a normal (USB) microphone? 
Maybe the stream audio is that bad ...

**Reply by tsiawd** [reply]
Im 90% sure something is wrong with the models or something. All the birds listed in the logs are from a different country.

**Reply by DMontgomery40** [reply]
When you set your frequency threshold and location, you can see every single bird that can qualify as being detected. So check that, if it feels like every bird is from halfway around the world, I'd say there's a good chance you've mixed up your longitude and latitude. And are therefore literally getting birds from the other side of the world, exactly the other side.  

![image](https://github.com/mcguirepr89/BirdNET-Pi/assets/130489651/99589403-007f-4bb5-8474-0c635af1ec45)

If you are in North America your latitiude should be a two digit number (30s or 40s) and longitude should be a three digit number. Ideally followed by a few decimal points. Two decimal points will obfuscate your location, if you're concerned about that, but still get you birds that are in the area. 

Additionally, if you signed up for a birdweather id, which you should absolutely do, your birdnet-pi station will be on the map at app.birdweather.com - and you can double verify you're in the right place. 

![image](https://github.com/mcguirepr89/BirdNET-Pi/assets/130489651/5613c68d-0f5c-4f82-a340-01440454a1ae)

AI models are far from perfect, this one included, but with your location set correctly, it's literally impossible for you to get birds from around the world no matter how bad the model is. 

Once you have that set correctly, get a usb mic, even a cheap one for like $7 off amazon will be better than most RTSP streams, unless you have a camera that allows you to pick your audio codec, sampling rate, etc. Most IP cams allow this, all my Amcrest cams do, but I would never use those for accurate detections. 

Pretty certain you mixed up lat/lon.  Also, make sure you're using the newer v2.4 model (choose that in the same place you choose your threshold).

**Comment by tsiawd** [comment]
I dont know what was going on but I think I installed another version off a different github  but I found a current one and now everything works as expected. It only took me 1/2 a day to figure out...
